web/src/beta/components/RadioGroup/RadioBox/index.tsx

```diff
@@ -23,7 +24,7 @@ const RadioBox: React.FC<Props> = ({ inactive, selected, value, onClick }: Props
       <RadioButton selected={isChecked} inactive={inactive}>
         {isChecked && <Checkmark selected={isChecked} inactive={inactive} />}
       </RadioButton>
-      <RadioText>{value}</RadioText>
+      <RadioText>{label ?? value}</RadioText>
```

Comment:
What we show for the radio button should never be the value I think. The label should be internationalized (and possibly capitalized depending on the language). Saved values will most definitely be keys, so can you update this component in that way?
So
- value -> key
- pass key in to onClick()
- use label only for RadioText
- Make key and label non-optional

Reply:
sorry but I am a little confused about the key and value , when you said value=> key , you mean only changing the name right ? , aaswe need separate value whatever its name and sometimes it isdifferent that the label for example the case of data source manager

Reply:
About passing the key or value to onclick isn't the same as it is now as we have value param and we already passed it to onclick

Reply:
yeah value -> key means name change. Let me know if anything else doesn't make sense

Reply:
ah, but actually I don't thing we should do that first of all is a reserved keyword and doesn't make sense to be prop in a component , for example radio box is used like this ` <RadioGroupContainer layout={layout}>
      {currentOptions.map((option, index) => (
        <RadioBox
          key={`${option.label}-${key}-${index}`}
          value={option.value}
          selected={option.selected}
          label={option.label}
          onClick={() => handleRadioChange(option.value)}
        />
      ))}
    </RadioGroupContainer>` potential confusion

Reply:
anyway I renamed it to be keyValue